roles/confluent.kafka_broker/tasks/restart_kafka.yml

```diff
@@ -1,8 +1,10 @@
 ---
```

Comment:
this file is duplicating code, can you make use of variable for the service name